abi/cbor_gen.go

```diff
@@ -163,3 +163,38 @@ func (t *SectorID) UnmarshalCBOR(r io.Reader) error {
 	}
 	return nil
 }
+
+var lengthBufAddrPairKey = []byte{128}
+
+func (t *AddrPairKey) MarshalCBOR(w io.Writer) error {
```

Comment:
This is going to encode/decode nothing unless you make the struct members public.

Reply:
Oof, thank you, fixed